firestore/transaction.go

```diff
@@ -59,7 +62,35 @@ var ReadOnly = ro{}
 
 type ro struct{}
 
-func (ro) config(t *Transaction) { t.readOnly = true }
+func (ro) config(t *Transaction)                     { t.readOnly = true }
+func (ro) handleCommitResponse(r *pb.CommitResponse) {}
+
+// CommitResponse is a wrapper over pb.CommitResponse to avoid exposing proto directly.
```

Comment:
Document this without referring to protos. (Say what it actually is.)

Reply:
Done.